docs/poolRegistration.md

```diff
@@ -124,31 +187,31 @@ cardano-cli shelley transaction build-raw \
 --tx-in 73aa1b60a8e32bae39a69b509e03f4b45f297817abb0e29d3eed92ece9dc1bbe#0 \
 --tx-out $(cat operator/payment.addr)+8000000 \
 --ttl 15770560 \
---fee 2000000 \
+--fee 194685 \
 --out-file tx.raw \
 --certificate-file pool-registration.cert
 ```
 
-## Sign the transaction
+## Create transaction witnesses
 
-This includes creating witnesses with the operator, pool and for all included pool owners
+This includes creating witnesses with the operator, pool and for all included pool owners. Only pool owners can provide their witness from a hardware wallet.
```

Comment:
Maybe explicitly clarify that the other keys (cold keys/spending keys) need to be managed in the cli.